src/Microsoft.ML.ImageAnalytics/ImagePixelExtractor.cs

```diff
@@ -478,15 +471,27 @@ private VectorDataViewType[] ConstructTypes()
     }
 
     /// <summary>
-    /// <see cref="IEstimator{TTransformer}"/> that extracts the pixels of the image into a vector.
-    /// This vector can further be used as a feature to the algorithms.
+    /// <see cref="IEstimator{TTransformer}"/> for the <see cref="ImagePixelExtractingTransformer"/>.
     /// </summary>
     /// <remarks>
-    /// Calling <see cref="IEstimator{TTransformer}.Fit(IDataView)"/> in this estimator, produces an <see cref="ImagePixelExtractingTransformer"/>.
-    /// <seealso cref="ImageEstimatorsCatalog.ExtractPixels(TransformsCatalog, ImagePixelExtractingEstimator.ColumnOptions[])" />
-    /// <seealso cref="ImageEstimatorsCatalog.ExtractPixels(TransformsCatalog, string, string, ColorBits, ColorsOrder, bool, float, float, bool)" />
-    /// <seealso cref="ImageEstimatorsCatalog"/>
+    /// <format type="text/markdown"><![CDATA[
+    ///
+    /// ###  Estimator Characteristics
+    /// |  |  |
+    /// | -- | -- |
+    /// | Does this estimator need to look at the data to train its parameters? | No |
+    /// | Input column data type | <xref:System.Drawing.Bitmap> |
+    /// | Output column data type | Vector of known size of <xref:System.Single> or <xref:System.Byte> |
+    ///
```

Comment:
>  /// [](start = 2, length = 5)

please add: |Required NuGet in addition to Microsoft.ML| the nuget name |

also to other image transforms in the PR